chapter_computational-performance/async-computation_vn.md

```diff
@@ -95,7 +95,7 @@ Hence it is ony possible to parallelize operations that do not depend on each ot
 -->
 
 Nhìn chung, MXNet có front-end cho phép tương tác trực tiếp với người dùng thông qua Python, cũng như back-end được sử dụng bởi hệ thống nhằm thực hiện nhiệm vụ tính toán.
-Back-end có các luồng xử lý riêng liên tục tập hợp và thực hiện các tác vụ trong hàng đợi.
+Back-end có các luồng xử lý riêng liên tục tập hợp và thực thi các tác vụ trong hàng đợi.
 Chú ý rằng, back-end cần có khả năng theo dõi quan hệ phụ thuộc giữa nhiều bước khác nhau trong đồ thị tính toán để có thể hoạt động.
 Do đó ta chỉ có thể song song hoá các thao tác không phụ thuộc lẫn nhau.
```

Comment:
```suggestion
Do đó ta chỉ có thể song song hoá các phép toán không phụ thuộc lẫn nhau.
```

Reply:
trong ngữ cảnh lập trình mình prefer dịch `operation` theo `thao tác` > `phép tính` > `phép toán`.

Reply:
mình thấy ở đây đang nói về tính toán thì để vậy cũng được nhỉ

Reply:
btw ở bên bản eng mới xóa luôn đoạn này rồi

Reply:
nhờ @lkhphuc xóa giúp đoạn này nhé